.github/workflows/tests.yml

```diff
@@ -127,18 +127,21 @@ jobs:
         shell: bash
         run: ./scripts/build.sh /tmp/e2e-test/avalanchego/plugins/srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy
       - name: Start prometheus
+        if: secrets.PROMETHEUS_ID != ''
```

Comment:
hmm this file is not working https://github.com/ava-labs/subnet-evm/actions/runs/8471478509/workflow
I think it should be ${{ secrets.PROMETHEUS_ID }}